src/chapters/06-interactive-components.md

```diff
@@ -2,13 +2,24 @@
 ember server
 ```
 
+In this chapter, you will learn about:
+* Adding behavior to components with classes
+* Accessing instance variable states from templates
+* Managing state with tracked properties
+* Using conditionals syntaxes in templates
+* Responding to user interaction with actions
+* Invoking element modifiers
+* Testing user interactions
+
+## Adding Behavior to Components with Classes
+
 So far, all the components we have written are purely *[presentational][TODO: link to presentational]* &mdash; they are simply reusable snippets of markup. That's pretty cool! But in Ember, components can do so much more.
 
 Sometimes, you want to associate some *[behavior][TODO: link to behavior]* with your components so that they can do more interesting things. For example, `<LinkTo>` can respond to clicks by changing the URL and navigating us to a different page.
 
 Here, we are going to do just that! We are going to implement the "View Larger" and "View Smaller" functionality, which will allow our users to click on a property's image to view a larger version, and click on it again to return to the smaller version.
 
-In other words, we want a way to _toggle_ the image between one of the two *[states][TODO: link to states]*. In order to do that, we need a way for the component to store two possible states, and to be aware which states it is currently in.
+In other words, we want a way to _toggle_ the image between one of the two *[states][TODO: link to states]*. In order to do that, we need a way for the component to store two possible states, and to be aware of which states it is currently in.
```

Comment:
I think this actually want to say "which state it is currently in" (singular "state")